src/templates/zh-CN/2020/methodology.html

```diff
@@ -0,0 +1,475 @@
+{% extends "base/2019/methodology.html" %}
```

Comment:
This has now moved to reflect the fact it's no longer just for 2019 (This is what is causing your test failure btw):

```suggestion
{% extends "base/methodology.html" %}
```

Reply:
Wow,thank you very much!